### PR TITLE
[New map] NSV Hammerhead!

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

The NSV Hammerhead is an entirely new, state of the art map designed around fun multiz interactions and verticality! It aims to join Atlas in the rotation for a classic lowpop map, but while also providing a great midpop option like Gladius. Similarly to Tycoon, it's also a fantastic highpop option, much like the Galactica, it can also support populations of up to 100!

The map has been rigorously tested over several months for quality assurance purposes, to ensure that it's released in its best state possible.

## Why It's Good For The Game

Deck 1: A spinally mounted corridor to allow for easy access to core systems
image

Deck 2: The entire ship

image

The NSV Hammerhead is classed as a heavy cruiser. A large, unwieldy ship specialising in heavy weapons. It's equipped with a standard NAC / VLS / Gauss / PDC loadout in a unique, compact form factor! Due to its size, this ship uses flak to assist with point defense :)
image

Special thanks to Francinum for his amazing work in this collaboration :)) 

## Changelog
:cl: Francinum & Kmc2000
add: Added a brand new map, NSV Hammerhead! You'll see it around lowpop, medium pop, highpop, tidepop, infinite pop, at the heat death of the universe, and byond even that which we may comprehend with our mortal minds.
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
good pr

**Comment by francinum** [comment]
based

**Comment by RemervKeberb** [comment]
Very wise PR

**Comment by Pontenerd** [comment]
THE HAMMER STRIKES ANOTHER BLOW, SHE FLIES ONCE MORE

**Comment by athena413** [comment]
YOOOO

**Comment by retlaw34** [comment]
its been 3000 years

**Comment by AnCopper** [comment]
Hammerhead? Haven't heard that name in a while, heh.

**Comment by Demoxide** [comment]
Speedmerge this! Come on!

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
Replace nwp with amW on line 39285 of Hammerhead1.dmm, it contains a memetic hazard.

File: config/maps.txt [hunk 1]
```diff
@@ -37,7 +37,7 @@ endmap
 #map Vago
 endmap
 
-#map hammerhead
+map hammerhead
```
[the remark this answers is not in the document]
Comment (Kmc2000) [reply config/maps.txt:40]: no bokkie you can never escape the hammerhead

**Comment by Bokkiewokkie** [comment]
> From the discord discussion:
> 
> The new spinal hallway does no good if it doesn't feature some fast way of transportation, it will just compound the fundamental hallway issue it's already experiencing. I do recommend throwing in a pair of conveyors or a pair of bluespace tiles for fast lane transportation on the top deck. It is our longest ship
> 
> If you do add conveyors or bluespace tiles, please lengthen it down to engineering! I spend all my time there

Obviously the correct solution to all our problems is adding the people tubes in maint that were part of the original Hammerhead design, I'm kind of disappointed to see you guys left those out in the final release.

**Pull request closed without merging**